frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/portlets/components/ComponentPortletUtils.java

```diff
@@ -69,6 +65,9 @@ public static void updateReleaseFromRequest(PortletRequest request, Release rele
                         release.setReleaseIdToRelationship(new HashMap<String, ReleaseRelationship>());
                     updateLinkedReleaesFromRequest(request, release.releaseIdToRelationship);
                     break;
+                case CLEARING_STATE:
+                    // skip setting CLEARING_STATE. it is supposed to be set only programmatically, never from user input.
```

Comment:
understand the point, but "sw360 is designed to work with fossology". Also, it could be OK, my points:
* clearing state is not really fossology dependent, but tool dependent.
* if there is a new tool, integration should be done accordingly (see also the ticket #128 )
* if there is no tool, no status shall be displayed there
* for the manual state, there is mainline state
does this sound acceptable?